unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/CylinderShape/Handler/ECSCylinderShapeComponentHandler.cs

```diff
@@ -42,11 +47,21 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBCyl
 
         private void GenerateRenderer(Mesh mesh, IParcelScene scene, IDCLEntity entity, PBCylinderShape model)
         {
+            GenerateHolder(entity);
             meshesInfo = ECSComponentsUtils.GenerateMeshInfo(entity, mesh, entity.gameObject, model.Visible, model.WithCollisions, model.IsPointerBlocker);
```

Comment:
shouldn't it be:
```suggestion
            meshesInfo = ECSComponentsUtils.GenerateMeshInfo(entity, mesh, meshHolderGameObject, model.Visible, model.WithCollisions, model.IsPointerBlocker);
```
?
I saw other components also not changing this arg

Reply:
Good catch!